crt_portal/cts_forms/tests.py

```diff
@@ -255,23 +255,28 @@ def test_workplace_primary_complaint_routing(self):
 
         SAMPLE_REPORT['primary_complaint'] = 'workplace'
         test_report = Report.objects.create(**SAMPLE_REPORT)
+
         test_report.protected_class.add(immigration[0])
         test_report.save()
         self.assertTrue(test_report.assign_section() == 'IER')
+
         test_report.protected_class.remove(immigration[0])
         test_report.protected_class.add(language[0])
         test_report.save()
         self.assertTrue(test_report.assign_section() == 'IER')
+
         test_report.protected_class.remove(language[0])
         test_report.protected_class.add(origin[0])
         test_report.save()
         self.assertTrue(test_report.assign_section() == 'IER')
+
         test_report.protected_class.add(language[0])
         test_report.save()
         self.assertTrue(test_report.assign_section() == 'IER')
+
         test_report.protected_class.add(disability[0])
         test_report.save()
-        self.assertTrue(test_report.assign_section() == 'IER')
+        self.assertTrue(test_report.assign_section() == 'DRS')
```

Comment:
This is a change since we got the IER ticket, I think it is correct but wanted to get confirmation. I started a thread here: 
https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/73#issuecomment-571819555